src/main/io/serial_cli.c

```diff
@@ -317,6 +317,7 @@ const clivalue_t valueTable[] = {
     { "frsky_default_longitude",    VAR_FLOAT  | MASTER_VALUE,  &masterConfig.telemetryConfig.gpsNoFixLongitude, -180.0, 180.0 },
     { "frsky_coordinates_format",   VAR_UINT8  | MASTER_VALUE,  &masterConfig.telemetryConfig.frsky_coordinate_format, 0, FRSKY_FORMAT_NMEA },
     { "frsky_unit",                 VAR_UINT8  | MASTER_VALUE,  &masterConfig.telemetryConfig.frsky_unit, 0, FRSKY_UNIT_IMPERIALS },
+    { "Hott_alarm_sound_period",    VAR_UINT8  | MASTER_VALUE,  &masterConfig.telemetryConfig.HottAlarmSoundPeriod, 0, 120 },
```

Comment:
well the alarm sound comes each "hott_alarm_sound_period" in s, so this is a real "period" and not a duration, unless there's a subtle english equivalence (?)